packages/client/hmi-client/src/components/widgets/tera-asset.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<main id="Top">
+		<header>
+			<section>
+				<span class="asset-form">{{ assetForm }}</span>
```

Comment:
It's what's right above the `h4` in a Model it could say 'Petrinet', in a Dataset it could say 'Simulation run'. For a document it shows the journal name which is a bit off... `assetForm` was the name I came up with, I guess we could think of something better

Reply:
Yeah, maybe subtitle would be better.

Reply:
Subtitle I feel belongs below the big header. What do you think about using overline 
![image](https://user-images.githubusercontent.com/28237258/231878781-bea25977-5c32-4202-88bd-83577113eeb2.png)